src/background.js

```diff
@@ -171,6 +197,12 @@ async function handlePopupMessage(message) {
 			if (browser.tabs.print) {
 				return browser.tabs.print();
 			}
+		} case "fullscreen": {
+			return browser.windows.getCurrent().then(win =>
+				browser.windows.update(browser.windows.WINDOW_ID_CURRENT, {
+					state: win.state === "fullscreen" ? "normal" : "fullscreen"
+				})
+			);
```

Comment:
Also, we use [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) here instead of [`Promise.then()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then).